### DISCUSSION TITLE
Story size

### DISCUSSION THREAD

**Discussion by justified1gmail** [Videogamers0/StoryManager#9] [Ideas]
Any chance of displaying story size in the sidebar?

**Comment by Videogamers0** [comment]
Sure, what did you have in mind? The word count or the filesize?

**Reply by justified1gmail** [reply]
The Literotica story Info pane shows words.  

I believe page count may be more user friendly, but any metric (words, characters, pages, file size) would be useful.

**Reply by Videogamers0** [reply]
Added page count and word count display settings to the filtering dropdown (top-right of stories list): f2dec3aa13529e89fb295fb730cf2364a0f1960d

[Version 1.0.4](https://github.com/Videogamers0/StoryManager/releases) now has these features
